runtime/parachains/src/paras_inherent.rs

```diff
@@ -2016,83 +1973,49 @@ mod tests {
 				})
 				.collect::<Vec<_>>();
 
-			{
-				assert_matches!(
-					sanitize_backed_candidates::<Test, _, true>(
-						relay_parent,
-						backed_candidates.clone(),
-						has_concluded_invalid,
-						scheduled
-					),
-					Ok(sanitized_backed_candidates) => {
-						assert_eq!(backed_candidates, sanitized_backed_candidates);
-					}
-				);
-				assert_matches!(
-					sanitize_backed_candidates::<Test, _, false>(
-						relay_parent,
-						backed_candidates.clone(),
-						has_concluded_invalid,
-						scheduled
-					),
-					Ok(sanitized_backed_candidates) => {
-						assert_eq!(backed_candidates, sanitized_backed_candidates);
-					}
-				);
-			}
+			// happy path
+			assert_eq!(
+				sanitize_backed_candidates::<Test, _>(
+					relay_parent,
+					backed_candidates.clone(),
+					has_concluded_invalid,
+					scheduled
+				)
+				.len(),
+				2
```

Comment:
Verify the content is identical.

Reply:
e1c32e6a82b85010dc9dfc25740b4cbdaed7be8a